code/modules/reagents/chemistry/reagents/toxin_reagents.dm

```diff
@@ -1109,7 +1109,7 @@
 /datum/reagent/toxin/acid/nitracid
 	name = "Nitric Acid"
 	description = "Nitric acid is an extremely corrosive chemical substance that violently reacts with living organic tissue."
-	color = "#5050FF"
+	color = "#007718"
```

Comment:
Nitric acid should be reddish-brown

Reply:
$\color{#669900}&#9724;$ #669900 - is a green mixed with brown/orange - that is kinda olive